src/test/java/teammates/logic/core/FeedbackSessionsLogicTest.java

```diff
@@ -700,6 +704,182 @@ public void testUpdateFeedbackSession_shouldAdjustEmailSendingStatusAccordingly(
                 typicalSession.getFeedbackSessionName(), typicalSession.getCourseId()).isSentPublishedEmail());
     }
 
+    @Test
+    public void testUpdateFeedbackSessionsInstructorDeadlinesWithNewEmail() {
```

Comment:
Maybe the test case name can describe the input and the output, say `updateInstructorEmail_transferDeadlinesToNewEmail`?

Reply:
I think the test method name gets a bit too long if I add it to the end:

```
public void testUpdateFeedbackSessionsInstructorDeadlinesWithNewEmail_updateInstructorEmail_transferDeadlinesToNewEmail()
```

Or do you mean replacing it completely? Wouldn't this be unconventional?

```
public void testUpdateInstructorEmail_transferDeadlinesToNewEmail()
```

Or how about using something like `______TS("Update instructor email; transfers deadlines to new email.")`?

Reply:
I think TS will do. Indeed it isn't too conventional to just replace as such.